libyul/backends/evm/EVMDialect.cpp

```diff
@@ -186,6 +186,12 @@ std::set<std::string, std::less<>> createReservedIdentifiers(langutil::EVMVersio
 		"setimmutable",
 		"loadimmutable",
 	};
+
+	if (_eofVersion.has_value())
+		reserved += std::vector<std::string>{
+			"auxdataloadn",
+		};
```

Comment:
@clonker I think we should refactor this so that reserved identifiers automatically contain all builtin names from the current dialect. Then remove the ones that are exceptions. What do you think? It would make this kind of mistake harder to make.

Reply:
Yes, absolutely. In my mind it is more likely that we add new reserved builtins compared to the amount of times we would be adding more exceptions.